CodeEdit/Features/InspectorSidebar/InspectorSidebarTabBar.swift

```diff
@@ -98,35 +90,33 @@ struct InspectorSidebarTabBar: View {
     }
 
     private func makeIcon(
-        named: String,
-        title: String,
-        id: Int,
+        tab: InspectorTab,
         scale: Image.Scale = .medium,
         size: CGSize
     ) -> some View {
         Button {
-            selection = id
+            selection = tab.id
         } label: {
-            getSafeImage(named: named, accessibilityDescription: title)
+            getSafeImage(named: tab.systemImage, accessibilityDescription: tab.title)
                 .font(.system(size: 12.5))
-                .symbolVariant(id == selection ? .fill : .none)
+                .symbolVariant(tab.id == selection ? .fill : .none)
                 .frame(
                     width: position == .side ? 40 : 24,
                     height: position == .side ? 28 : size.height,
                     alignment: .center
                 )
-                .help(title)
-                .onDrag {
-                    if let index = icons.firstIndex(where: { $0.imageName == named }) {
-                        draggingItem = icons[index]
-                    }
-                    return .init(object: NSString(string: named))
-                } preview: {
-                    RoundedRectangle(cornerRadius: .zero)
-                        .frame(width: .zero)
-                }
+                .help(tab.title)
+//                .onDrag {
+//                    if let index = icons.firstIndex(where: { $0.imageName == named }) {
+//                        draggingItem = icons[index]
+//                    }
+//                    return .init(object: NSString(string: named))
+//                } preview: {
+//                    RoundedRectangle(cornerRadius: .zero)
+//                        .frame(width: .zero)
+//                }
```

Comment:
Will this be reused/reenabled at some point?

Reply:
Yes, it's meant to be reenabled later on. This has been discussed with @austincondiff on Discord.